src/process_helpers.c

```diff
@@ -5,6 +5,15 @@
 #include <sys/wait.h>
 #include <signal.h>
 
+#ifndef __linux
```

Comment:
Rather than mucking around with these signals  we should just remove them instead. I added a rather full list of signals when I added the process class, but these I shouldn't have added at all so let's just remove them instead from process.